tapchannel/commitment.go

```diff
@@ -707,6 +707,24 @@ func CreateAllocations(chanState *channeldb.OpenChannel, ourBalance,
 			allocType = CommitAllocationHtlcIncoming
 		}
 
+		dustLimit := chanState.LocalChanCfg.DustLimit
```

Comment:
This can be done at the top of the `CreateAllocations` function (after where we set `leaseExpiry`), then it can be re-used for both closures.